minkasi/tools/presets_by_source.py

```diff
@@ -351,25 +356,63 @@ def get_bad_tods(name, ndo=False, odo=False):
             ]  #
             addtag = "svprods_ODO_"
 
-    if name == "moo1142" or name == "mooj1142":
+    elif name == "moo1142" or name == "mooj1142":
+        # definitely bad TODs
         bad_215_04 = [
-            "Signal_TOD-AGBT18B_215_04-s" + str(ss) + ".fits"
-            for ss in [52, 53, 54, 86, 87, 88, 89, 91, 92, 93]
-        ]
-        bad_019_02 = [
-            "Signal_TOD-AGBT20A_019_02-s" + str(ss) + ".fits" for ss in [28, 43]
+            "Signal_TOD-AGBT18B_215_04-s" + str(ss) + ".fits" for ss in [88, 89, 91, 92, 93, 94]
         ]
         bad_005_02 = [
-            "Signal_TOD-AGBT23B_005_02-s" + str(ss) + ".fits" for ss in [17, 39]
+            "Signal_TOD-AGBT23B_005_02-s" + str(ss) + ".fits" for ss in [19, 20, 35, 36, 37]
         ]
-        bad_005_08 = [
-            "Signal_TOD-AGBT23B_005_08-s" + str(ss) + ".fits"
-            for ss in [14, 16, 17, 18, 26, 27, 28, 29]
+        bad_005_04 = [
+            "Signal_TOD-AGBT23B_005_04-s" + str(ss) + ".fits" for ss in [8, 9, 10]
         ]
-        bad_tod.extend(bad_215_04)
-        bad_tod.extend(bad_019_02)
+        bad_005_05 = [
+            "Signal_TOD-AGBT23B_005_05-s" + str(ss) + ".fits" for ss in [27, 28]
+        ]
+        bad_005_06 = [
+            "Signal_TOD-AGBT23B_005_06-s" + str(ss) + ".fits" for ss in [10, 11]
+        ]
+        bad_tod = bad_215_04
```

Comment:
This will overwrite anything above in `bad_tod` which is not desired behavior. Ideally the initialization of `bad_tod` should not be in an `if` statement.

Reply:
I will revert the code for this script back to how it was so this will no longer be an issue.